src/components/MarkdownProvider/components/PropSheets.tsx

```diff
@@ -9,25 +9,17 @@ import React from 'react';
 import { css } from 'styled-components';
 import { ComponentDoc } from 'react-docgen-typescript';
 import { getColor } from '@zendeskgarden/react-theming';
-import { MD, Paragraph } from '@zendeskgarden/react-typography';
+import { Paragraph } from '@zendeskgarden/react-typography';
 import { Table, Head, Body, HeaderRow, HeaderCell, Row, Cell } from '@zendeskgarden/react-tables';
-
-import { IPackage } from './PackageDescription';
 import { StyledH3 } from './Typography';
 
-export const PropSheets: React.FC<{ data: ComponentDoc[]; reactPackage: IPackage }> = ({
-  data,
-  reactPackage
-}) => {
+export const PropSheets: React.FC<{ propSheets: ComponentDoc[] }> = ({ propSheets }) => {
```

Comment:
Could you go into depth on the prop rename for `data -> propSheets`? With the name of the component the usage will now be `<PropSheets propSheets={...} />` which feel repetitive

Reply:
As Henry and I were working through, the naming of this section was confusing to understand so this hoped to address that a bit. The prop was `data` on `PackageDescription` but it's grabbing the package information. But then on `PropSheets` component the `data` property was grabbing something different and reactPackage is grabbing package information. Open to changes, but the hope was to make it more straightforward to read. 

```
<PackageDescription data={props.data.mdx.package} />
<PropSheets data={props.data.mdx.propsSheets} reactPackage={props.data.mdx.package} />
```

Reply:
I noticed that `propSheets` was passed as "data", but `package` is passed as "reactPackage". Why would one prop be named "data" but not the other? For example:

```tsx
<Configuration reactPackage={props.data.mdx.package} data={props.data.mdx.propsSheets} />
```

My suggestion was what Ross currently has in this PR, or:

```tsx
<Configuration data={...props.data.mdx.package, ...props.data.mdx.propsSheets} />
```

Let us know what you think @austin94. I'm down with reverting it to what you had originally 👍.

Reply:
A single data prop makes sense to me. Feel free to go with whatever feels best.